_dprhtml/features/navigation/init.js

```diff
@@ -59,14 +59,21 @@ class NavigationTabViewModel {
     this.para = ko.observable('');
 
     this.places = ko.observableArray();
+    this.locPlace = ko.observableArray();
```

Comment:
can we please give it a more descriptive name?

Reply:
sectionPlace